specification/network/resource-manager/readme.md

```diff
@@ -28,7 +28,7 @@ These are the global settings for the Network API.
 title: NetworkManagementClient
 description: Network Client
 openapi-type: arm
-tag: package-2021-03
+tag: package-2021-02-preview
```

Comment:
@yanfa317 Because you already has 2021-03 table version, we can not change the default tag back to 2021-02-preview a previous preview version. if anything, we can only allow change into 2021-04-preview and that still depends on 2021-04-preview contains all the non-deprecated features in 2021-03.

Reply:
@qiaozha, we are trying to release an individual preview version of SDK based on  2021-02-preview, since our service is only exist in folder 2021-02-01-preview, so we can't use 2021-04-preview or 2021-03. Our SDK package will be separated from normal network release Package. And Network team will change the tag back when they do next round of monthly release

Reply:
@yanfa317 are you only generating for .NET SDK? If so, to limit the impact, you can modify the `generate.ps1` file in your .NET SDK src folder to specify a tag with `-ConfigFileTag`, like this one did: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/mysql/Microsoft.Azure.Management.MySQL/src/generate.ps1

Reply:
Thanks @fengzhou-msft , I am trying it. I have an one more question here. So what if we need python sdk also? Cli team are helping us to create the cli package, but I am not sure if they can provide sdk for us. Thanks!